### ISSUE TITLE
The log in button is not fully accessible to users using a screen reader

### ISSUE THREAD

**Issue by Andrea-Guevara** [DSpace/dspace-angular#3249]
## Describe the bug
When the "log in" button is disabled, it is not possible to reach it from the keyboard, nor is it possible to inform users using the screen reader that the button is disabled.

## To Reproduce
1. Go to the login area.
2. Navigate via the keyboard using a screen reader.
3. See that the "log in" button, when disabled, is not accessible via keyboard navigation.
4. See that the button is not accessible to users using a screen reader.

## Expected behavior
The "log in" button, whether enabled or disabled, needs to be accessible to users who use the screen reader.

**Comment by artlowel** [comment]
Atmire would like to claim this issue. We'll also look for other disabled buttons to fix, as they'll have the same problem

**Comment by Andrea-Guevara** [comment]
Good afternoon @tdonohue! We at Neki-it have carried out some tests and have come up with a possible solution to this problem. Initially, the accessibility problem is fixed only on the login button. But as @artlowel said, this is a problem with all buttons disabled.
